FormValidator/index.html

```diff
@@ -80,14 +81,13 @@ <h1> Form Validator</h1>
         const data = new FormData(this);
         for (const entry of data) {
             if(entry[1].length === 0) {
-                console.log(entry[0])
                 $result.innerText = 'All field should not empty';
                 $result.classList.add('text-red');
                 return;
             }
         }
 
-        if( $name.value.length > 15 ) {
+        if( $name.value.length > NAME_MAXLEN) {
             $result.innerText = 'Name should not longer than 15 charaters';
```

Comment:
Suggestion: move the text to the separate file for easy maintenance of the content